CI/Dockerfile

```diff
@@ -41,6 +41,7 @@ RUN apt-get -y update; \
                        python3 \
                        python3-pip \
                        python3-setuptools \
```

Comment:
setuptools and packaging are mentioned in the pyproject.toml of pymoab so ideally they would have been installed by python during the instal process.

I see adding them to the docker works which is great but I would be keen to understand why they are not being installed automatically like other build dependencies